catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -432,6 +432,14 @@
           "description": "Variance of a column.",
           "type": "number"
         },
+        "earliestTime": {
```

Comment:
@nna077 Can you adde some details what are you trying to solve here?

Reply:
Yes sure, These fields help find the latest time and earliest time in a Date/Time/Timestamp column. Say for example a service such as Trino has a table that contains three rows and a single column field which is of type: (Date). Row 1: 2020-01-01 Row 2: 2020-02-02 Row 3: 2020-03-03,  earliestTime = 2020-01-01 latestTime = 2020-03-03. This stat would be made available in the Profiler Tab. I have also attached a UI screenshot for clarification. 
![image](https://user-images.githubusercontent.com/107413937/177064529-8274f407-7e32-49cf-bf68-0f5e62917645.png)